matchzoo/engine/base_preprocessor.py

```diff
@@ -57,28 +58,28 @@ def fit_transform(self, inputs: list, stage: str) -> datapack.DataPack:
 
     def _make_output(
         self,
-        output: list,
-        mapping: pd.DataFrame,
+        relation: typing.Union[list, np.ndarray],
```

Comment:
relation is pandas?

Reply:
Here, the `_make_output` function is to construct a `DataPack` object based on 'relation & content & context & columns'. The construction function of `DataPack` only accept 'list or np.ndarray' as input for 'relation'. Thus, I do not use pandas for `relation`. In fact, the preprocess module transform a datapack (from segmendation) to a new datapack (by _make_output). During the transformation, the relation often keeps the same. Thus, a more elegant way is to re-assign the content rather than re-construct a datapack. What do you think ?